<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegInfoForList</class>
 <widget class="QDialog" name="RegInfoForList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>650</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Подтвердите, что Вы администратор</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aund">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>710</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: rgb(100, 95, 94);
	
	color: rgb(229, 223, 215);
	border-radius: 3px;
</string>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>720</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>693</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aund_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>670</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(120, 0, 0);
color: rgb(229, 223, 215);
border-radius: 3px;
</string>
   </property>
   <property name="text">
    <string>Сброс всех устройств</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scroll">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>900</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#scroll
{
background-color: rgb(190, 190, 190);
border-style: outset;
border-width: 3px;
border-color: rgb(190, 191, 169);
background-image: url(:/res/img/dragon.png);
}
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>894</width>
      <height>544</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>600</y>
     <width>521</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	
	background-color: rgb(249, 249, 249, 80);
	opacity: 0.5;
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>600</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	
	background-color: rgb(249, 249, 249, 80);
	opacity: 0.5;
}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
